static/js/settings_users.js

```diff
@@ -251,13 +251,10 @@ exports.on_load_success = function (realm_people_data) {
         channel.del({
             url: '/json/users/' + encodeURIComponent(email),
             error: function (xhr) {
-                if (xhr.status.toString().charAt(0) === "4") {
-                    meta.current_deactivate_user_modal_row.find("button").closest("td").html(
-                        $("<p>").addClass("text-error").text(JSON.parse(xhr.responseText).msg)
-                    );
-                } else {
-                    meta.current_deactivate_user_modal_row.find("button").text(i18n.t("Failed!"));
-                }
+                var status = $("#organization-status").expectOne();
+                ui_report.error(i18n.t("Failed"), xhr, status);
+                var button = meta.current_deactivate_user_modal_row.find("button.deactivate");
+                button.text(i18n.t("Deactivate"));
```

Comment:
Set the button value to "Deactivate". No need to set it to "Failed", cause we are already showing error on top of page.